README.md

```diff
@@ -545,6 +545,12 @@ No configuration is needed.
 
 Visualizes rdfa.
 
+### (Experimental) RDFA edit and debug tools plugin
+
+Enables (experimental) RDFA edit and debug tools plugin. This plugin is visible in the side panel.
```

Comment:
Could do with a bit more content, though I guess we could just link to the editor docs: https://github.com/lblod/ember-rdfa-editor#experimental-a-new-approach-to-handle-rdfa-in-documents